pkg/compiler/convert_test.go

```diff
@@ -236,3 +238,118 @@ func TestInterfaceTypeConversion(t *testing.T) {
 	}`
 	eval(t, src, big.NewInt(1))
 }
+
+func TestConvert_Uint(t *testing.T) {
+	nums := []uint64{
+		0,
+		1,
+		127,
+		128,
+		255,
+		256,
+		1024,
+		1<<16 - 1,
+		1 << 16,
+		1<<32 - 1,
+		1 << 32,
+		1<<64 - 1,
+	}
+
+	testCases := []struct {
+		nums       []uint64
+		offset     int
+		typ        string
+		helperType string
+		endians    map[string]func([]byte, uint64)
+	}{
+		{
+			nums:       nums,
+			typ:        "uint64",
+			helperType: "Uint64",
+			endians: map[string]func([]byte, uint64){
+				"LE": binary.LittleEndian.PutUint64,
+				"BE": binary.BigEndian.PutUint64,
+			},
+		},
+		{
+			nums:       nums[:len(nums)-2],
```

Comment:
BTW, it's a bad test structure. You'll have to modify all testcases on `nums` list extension in future. At least use `nums[:constInt]`, here and below.